README.md

```diff
@@ -27,14 +27,27 @@ learning frameworks.
 ## Key Features
 
 - Support of various compression algorithms, applied during a model fine-tuning process to achieve a better performance-accuracy trade-off:
-  
+
+### Post-Training Compression Algorithms
+
+| Compression algorithm                                                       |PyTorch|TensorFlow|   ONNX   |       OpenVINO     |
+|:----------------------------------------------------------------------------| :---: | :---: |:--------:|:------------------:|
+| [Quantization](./docs/compression_algorithms/post_training/Quantization.md) | Supported | Supported |Supported| Preview |
+
+_Preview means that the work is in progress and NNCF does not guarantee the full functional support._
```

Comment:
```suggestion
_Preview means that this is a work in progress and NNCF does not guarantee the full functional support._
```

https://dictionary.cambridge.org/dictionary/english/work-in-progress

Reply:
not a fan of "that this is" structure